application/templates/deployment.yaml

```diff
@@ -303,4 +307,12 @@ spec:
       hostNetwork: {{ .Values.deployment.hostNetwork }}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
+      {{- if .Values.lifecycle }}
+      lifecycle:
+        {{- if .Values.lifecycle.preStop }}
```

Comment:
Let's just add here like this one:

```
lifecycle:
  {{ toYaml .Values.deployment.lifecycle | indent 8 }}
```

No need in separate `preStop` definition

Reply:
@kseniyashaydurova 
Done